batch/batch/front_end/templates/billing_projects.html

```diff
@@ -51,7 +51,7 @@ <h1>Billing Projects</h1>
       <form action="{{ base_path }}/billing_projects/create" method="POST">
         <input type="hidden" name="_csrf" value="{{ csrf_token }}">
         <tr>
-          <td><input type="text" name="billing_project"></td>
+          <td><input type="text" name="billing_project" autocomplete="off"></td>
```

Comment:
```suggestion
          <td><input type="text" name="billing_project" spellcheck="false" autocorrect="off"></td>
```

same thing here as well